docs/codeql/query-help/cpp-cwe.md

```diff
@@ -0,0 +1,3 @@
+## CWE coverage for C and C++
+
```

Comment:
For consistency, maybe you could an introduction and an **Overview** header here too? 🥇